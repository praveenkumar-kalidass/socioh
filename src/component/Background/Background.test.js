import React from 'react';
import { render } from '@testing-library/react-native';

import { Background1 } from './index';

describe('Background', () => {
  describe('Background1', () => {
    it('should match snapshot', () => {
      const container = render(<Background1 />);

      expect(container).toMatchSnapshot();
    });
  });
});
