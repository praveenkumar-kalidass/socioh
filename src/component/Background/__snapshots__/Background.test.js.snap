// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background Background1 should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "position": "absolute",
      },
    ]
  }
>
  <RNSVGSvgView
    bbHeight={1334}
    bbWidth={750}
    focusable={false}
    height={1334}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        Object {
          "flex": 0,
          "height": 1334,
          "width": 750,
        },
      ]
    }
    width={750}
  >
    <RNSVGGroup>
      <RNSVGCircle
        cx={83.75}
        cy={333.5}
        fill={4279943103}
        propList={
          Array [
            "fill",
          ]
        }
        r={93.75}
      />
      <RNSVGCircle
        cx={666.25}
        cy={1187.25}
        fill={4294892136}
        propList={
          Array [
            "fill",
          ]
        }
        r={166.75}
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
